src/Livewire/Settings/Profile.php

```diff
@@ -127,25 +115,38 @@ public function updatedAvatar(): void
 
     public function save(): void
     {
-        if (! user_can('my-profile.get') || auth()->id() !== $this->user['id']) {
-            return;
-        }
-
         $this->validate();
 
-        $notificationSettingsService = new NotificationSettingsService();
-        $notificationSettingsService->update(array_values($this->dirtyNotifications));
+        try {
+            UpdateUser::make($this->user)
```

Comment:
password confirmation is removed this way, update action rules before validating